packages/hint-validate-set-cookie-header/src/hint.ts

```diff
@@ -53,11 +53,18 @@ export default class ValidateSetCookieHeaderHint implements IHint {
             return value.replace(/(^")|("$)/g, '');
         };
 
+        /** Concat and unquote the strings after the first `=`. */
+        const splitByEqual = (splitResult: string[]): string[] => {
```

Comment:
Naming nit (since the split by equal is done before calling the function - also makes in more consistent with `normalizeAfterSplitByEqual` naming):
```suggestion
        const unquoteAfterSplitByEqual = (splitResult: string[]): string[] => {
```

Reply:
done!